src/modules/dom.js

```diff
@@ -0,0 +1,30 @@
+const restorauntObjects = 'https://www.themealdb.com/api/json/v1/1/categories.php';
```

Comment:
What do you think? :thinking:  There is if every need for us to avoid a global variable as much as we can. We could take this line to the  ``` getData ``` function and still achieve the same purpose.